File: AvitoStatistcService_Java/src/main/java/ru/espada/avito/statistic/model/request/GetStatisticAboutCategoryInPeriodRequest.java
package ru.espada.avito.statistic.model.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
// Запрос на получение предложения об изменении цены на продукт в зависимости от времени
public class GetStatisticAboutCategoryInPeriodRequest {

    private long categoryId;
    private long dateStart;
    private long dateEnd;

}

File: UserApiService_C#/AdminApiService/Models/Requests/GetStatisticAboutCategoryInPeriodRequest.cs
using Newtonsoft.Json;

namespace AdminApiService.Models.Requests;

public class GetStatisticAboutCategoryInPeriodRequest
{
    [JsonProperty("categoryId")]
    public long categoryId { get; set; }
    [JsonProperty("dateStart")]
    public long dateStart { get; set; }
    [JsonProperty("dateEnd")]
    public long dateEnd { get; set; } 
}